lib/mix/lib/mix.ex

```diff
@@ -661,6 +661,13 @@ defmodule Mix do
   This function can only be called outside of a Mix project and only with the
   same dependencies in the given VM.
 
+  The `MIX_INSTALL_RESTORE_PROJECT_DIR` environment variable may be specified.
```

Comment:
Similarly, let's remove the PROJECT from here:

```suggestion
  The `MIX_INSTALL_RESTORE_DIR` environment variable may be specified.
```

or maybe `MIX_INSTALL_RESTORE_PREVIOUS_DIR`.